Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -171,15 +182,11 @@ extension VideoRenderer {
         if let track = participant.track {
             log.debug("adding track to a view \(self)")
             self.add(track: track)
-            DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) {
+            DispatchQueue.global(qos: .userInteractive).asyncAfter(deadline: .now() + 0.01) { [weak self] in
```

Comment:
Why not update async without a delay?

Reply:
I can't remember now, but there were issues when it was done straight away. Should've put a comment back then. 😞